service/service.py

```diff
@@ -39,12 +39,43 @@
 # Import Flask application
 from . import app
 
+
+
+######################################################################
+# RETRIEVE A WISHLIST
+######################################################################
+@app.route("/wishlists/<int:wishlist_id>", methods=["GET"])
+def get_wishlists(wishlist_id):
+    """
+    Retrieve a single Wishlist
+    This endpoint will return a Wishlist based on it's id
+    """
+    app.logger.info("Request for wishlist with id: %s", wishlist_id)
+    wishlist = Wishlist.find(wishlist_id)
+    if not wishlist:
+        raise NotFound("Wishlist with id '{}' was not found.".format(wishlist_id))
+    return make_response(jsonify(wishlist.serialize()), status.HTTP_200_OK)
+
+
 ######################################################################
-# GET INDEX
+# ADD A NEW WISHLIST
 ######################################################################
-@app.route("/wishlists", methods=["GET"])
-def list_wishlists():
-    return 'Wishlist Home Page';
+@app.route("/wishlists", methods=["POST"])
```

Comment:
What happens if the request json is invalid?

Reply:
Do you mean DataValidationError in models.py?

Reply:
Yeah. The server will throw a Data Validation error, but the response will be 500 I believe.

Reply:
This will be included in issue #23 "add error handlers", the functions request_validation_error() and bad_request() could produce error message like "missing argument".

Reply:
We then need to prioritize that story I think. It will become a big task to handle errors for all the APIs in one ticket. What do you think?

Reply:
We can move it on the top. It may not be a big task, since the error handler code is given by sample project. I think Jesse is working on it now.

Reply:
Is there anything else I need to improve for this story?